loadtest/control/simplecontroller/actions.go

```diff
@@ -408,7 +409,8 @@ func (c *SimpleController) searchUsers() control.UserStatus {
 
 func (c *SimpleController) updateProfile() control.UserStatus {
 	userId := c.user.Store().Id()
-	userName := fmt.Sprintf("userName%d", c.id)
+	name := strings.TrimSuffix(c.user.Store().Username(), "-new")
+	userName := fmt.Sprintf("%s-new", name)
```

Comment:
This looks okay but this way we are changing just on the first iteration and after that always updating to the same username. This could possibly result in a server side optimization of not storing the new value at all if it's the same as the old one.

To avoid this we could either add a check and go back and forth from the original username and the one with the suffix or add a randomized fixed-length string before `-new`. Whatever you prefer.

Reply:
Sure, maybe we can add(replace) a random character every time we try yo update the `userName`.

Reply:
That works too :+1: